### DISCUSSION TITLE
Have fringe around dynamics block, like the one around source block.

### DISCUSSION THREAD

**Discussion by chemtov** [minad/org-modern#208] [Ideas]
Hi,

This is a really nice package and help org mode buffer to be clearer.

As $title says, is that something that could be done, ie having the fringe around dblocks ?

For instance that block:

```org-mode
#+BEGIN: columnview :hlines 1 :id local :match "real"
| ITEM  | FOO |   BAR | BAZ | ZOO |   ZUB |
|----------+-----+-------+-----+-----+-------|
| real1 | 729 |    55 |  22 |  47 | 30.18 |
| real2 | 563 | 28.55 |  21 |  42 | 20.28 |
#+TBLFM: @1$6=ZUB::$6=($3*4/$2) * 100;%.2f
#+END:
```

Note that `columnview` could be something another string depending on the dblock, see `org-dynamic-block-insert-dblock` for examples, but the user can define anything here. Also note that we might or might not have a TBLFM definition there.  The table itself is correctly rendered by the plugin.

Thanks again,
Regards,

**Comment by minad** [comment]
Good idea. Patches adding this are welcome.
